x-pack/plugins/alerting/server/rules_client/rules_client.ts

```diff
@@ -66,6 +66,23 @@ const fieldsToExcludeFromPublicApi: Array<keyof SanitizedRule> = [
   'activeSnoozes',
 ];
 
+export const fieldsToExcludeFromRevisionUpdates: Array<keyof Rule> = [
```

Comment:
Note: `Snooze` and `Mute` RuleClient methods have not been updated to include the revision increment logic, and so updates from those methods will not currently result in a revision increment. Do we want to include them?